pkg/metric/datum.go

```diff
@@ -14,6 +14,7 @@ type Datum struct {
 	Dimensions Dimensions   `json:"dimensions"`
 	Value      float64      `json:"value"`
 	Unit       StandardUnit `json:"unit"`
+	Kind       Kind         `json:"kind"`
```

Comment:
Just to be sure, do you intent to write the kind to elastic search when writing metrics?